### DISCUSSION TITLE
Is there no binding for the window's `alert()` method?

### DISCUSSION THREAD

**Discussion by winstxnhdw** [pyscript/pyscript#1497] [Q&A] [answered]
None of the following I tried has worked.

```python
# from js import alert
from pyodide import globals

alert = None
globals.set("alert", alert)
alert("test")
```

**Comment by JeffersGlass** [comment, marked as answer]
You can input the `alert` object directly from JavaScript and call it:

```python
from js import alert
alert("Hello, world!")
```

This works for me (Chromium and Firefox) - if it doens't work for you, I'd be curious what browser you're using.

**Reply by winstxnhdw** [reply]
You are right, it seems that my browser was not receiving the code change until I cleared the cache. Thank you.
